options/locale/locale_en-US.ini

```diff
@@ -314,6 +314,7 @@ team_no_units_error = Allow access to at least one repository section.
 email_been_used = The email address is already used.
 openid_been_used = The OpenID address '%s' is already used.
 username_password_incorrect = Username or password is incorrect.
+password_complexity =  Password does not pass complexity requirements. It must contain at least one lower case character, one upper case character,a number and a special character
```

Comment:
Given you've made this a configurable option this complexity statement could be completely incorrect.

Reply:
You are right, it is worth redoing the description